src/util-radix-tree.c

```diff
@@ -963,14 +961,16 @@ SCRadixNode *SCRadixAddKeyIPV6Netblock(uint8_t *key_stream, SCRadixTree *tree,
  *
  * \retval node Pointer to the newly created node
  */
-SCRadixNode *SCRadixAddKeyIPV4String(const char *str, SCRadixTree *tree, void *user)
+SCRadixNode *SCRadixAddKeyIPV4String(
```

Comment:
none of the users of this function seem to use the return code for anything other than a NULL check. So if we're going through the hassle of changing the function prototype, we can just switch to a `int` return code? Or perhaps a `SCError`. For some examples of that see https://github.com/OISF/suricata/pull/8812/commits/f855ee99a397da67cc37d32db15101690dadb003